libinstall/functions

```diff
@@ -1394,6 +1394,11 @@ prepare_apache_config() {
 
     # Prepare apache config
     ${CAT} << __EOT__ > $directory/centreon.apache.conf
+Header set X-Frame-Options: "sameorigin"
+ServerSignature Off
+ServerTokens Prod
+TraceEnable Off
```

Comment:
The apache documentation indicates that enabling TRACE method does not expose any security vulnerability.
https://httpd.apache.org/docs/2.4/en/mod/core.html#TraceEnable
Why should it be explicitly disabled?

Reply:
IMHO, the TraceEnable option should not be used in production mode, but only for debugging purposes.